server/app/controllers/fusor/api/v21/deployments_controller.rb

```diff
@@ -154,6 +155,33 @@ def log
       end
     end
 
+    def get_openshift_disk_space
+      # Openshift deployments need to know how much disk space is available on the NFS storage pool
+      # This method mounts the specifed NFS share and gets the available disk space
+      begin
+        nfs_address = @deployment.rhev_storage_address
+        nfs_path = @deployment.rhev_share_path
+        deployment_id = @deployment.id
+
+        cmd = "sudo safe-mount.sh '#{deployment_id}' '#{nfs_address}' '#{nfs_path}'"
+        status, _output = Utils::Fusor::CommandUtils.run_command(cmd)
+        if status != 0
+          raise 'Unable to mount NFS share at specified mount point'
```

Comment:
Not going to ask you do this, but you could always use unless instead of != and then put it on one line if you wanted.

```ruby
raise 'Unable to mount NFS share at specified mount point' unless status == 0
```